app/models/menu_items_menu.rb

```diff
@@ -4,7 +4,6 @@ class MenuItemsMenu < ActiveRecord::Base
 
   validates :daily_stock, presence: true, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
   validates :menu_item, presence: true
-  validates :menu, presence: true
```

Comment:
`m = FactoryGirl.build(:menu)`    //creates menu with id=nil
`mi = FactoryGirl.build(:menu_item)`
`m.menu_items << mi`               //adds  m.menu_items_menus[0].menu_id =nil
`m.menu_items_menus[0].daily_stock = 10`

`m.save`   ->  ROLLBACK
`m.errors.full_messages`    ->    ["Menu items menus menu can't be blank"] 
because `m.menu_items_menus[0].menu_id`   ->    nil

And it looks that this is how AA is handling it too.